app/static/src/scss/navbar.scss

```diff
@@ -0,0 +1,79 @@
+$govuk-border-colour: #b1b4b6;
+$govuk-link-colour: #1d70b8;
```

Comment:
Taken from https://github.com/communitiesuk/funding-service-design-assessment/pull/62 for the base template